sshcommand

```diff
@@ -4,6 +4,30 @@ set -e
 shopt -s nocasematch #For case insensitive string matching, for the first parameter
 
 SELF=`which $0`
+OSRELEASE=/etc/os-release
+
+f_print_os_name() {
+  sed -n 's#^NAME="\(.*\)"#\1#p' $OSRELEASE
+  return 0
+}
+
+f_adduser() {
+  local l_user l_platform
+
+  l_user=$1
+  l_platform="$(f_print_os_name)"
+  case $l_platform in
+    Debian|Ubuntu)
+      adduser --disabled-password --gecos "" $l_user
+      ;;
+    SLES|*)
```

Comment:
What I meant by my question is whether it works on the non-enterprise version, like opensuse.